examples/storage/createjob.go

```diff
@@ -103,7 +95,7 @@ func main() {
 		log.Fatalf("ListJobs: %s", err)
 	}
 
-	fmt.Printf("Number of jobs: %d\n", jobs.ResultSetSize)
+	fmt.Printf("Result set size: %d\n", jobs.ResultSetSize)
```

Comment:
**REALLY** want this output to display the results of the Job. I'm not sure if the library is broken or what. I haven't had the time to explore how results should properly return from Manta through the `GetOutput()` func. Would be awesome if that's something we can do here but I'm ok with doing it later.